packages/clay-color-picker/src/Custom.tsx

```diff
@@ -80,6 +80,8 @@ interface IProps {
 	 */
 	colors: Array<string>;
 
+	disablePalette?: boolean;
```

Comment:
Could we try to use the positive version of this API instead of a negative one? I find it easier to reason with than having to use a double-negative.

There's some rationale for this in [API Names and Begging the Negatives](https://ariya.io/2016/04/api-names-and-begging-the-negatives) which is also covered as part of [Fluent 2014: Design Strategies for JavaScript API](https://www.youtube.com/watch?v=y7RrP_fOYUQ)

@wincent, have we covered this in the past? Tried to check for an "API" good practices in guidelines but couldn't find one...

Reply:
> @wincent, have we covered this in the past?

It occasionally comes up in code review but I haven't seen it often enough to prompt me adding it to the guidelines, but maybe we should.

From ["Remove Double Negative"](https://refactoring.com/catalog/removeDoubleNegative.html):

> Double negatives are often frowned on by mavens of natural language ... double negatives are just plain confusing. So kill them on sight.

In my experience, _most_ of the time things read better if you avoid negation. The negation is fine until you need to negate it.... (eg. `!disablePalette`, which reads as "don't disable palette", which is a mouthful). In JS because we can easily have things like default params it is generally a good to favor the positive form, because you can make the dominant case (enabled) implicit and low friction:

```
function foo({enablePalette = true} = {}) {
  console.log(enablePalette);
}
```

Reply:
@jbalsas yeah that makes sense. I think the rationale for the `disable*` props probably stems from the `disabled` attr on html elements. I'm not tied to it though, so I'll go ahead and change it to a positive form and default the value to true.

On the naming, I don't think we typically use `enable` as a prefix anywhere. I think we would use `palette` or maybe `showPalette`(positive form 🤷‍♂️)? Any strong preferences for this?

Reply:
No strong prefs but `showPalette` sounds good to me.

Alternatives considered by discarded:

- `letYourPaletteShow`
- `paletteLivesMatter`
- `paletteOrNotToPaletteThatIsTheQuestionAndIThinkYouKnowTheAnswer`

Reply:
> letYourPaletteShow

😂  😂  😂  this one got me good

Reply:
🤣🤣🤣

showMeThePALETTE!!

Reply:
> showMeThePALETTE!!

![sick-reference](https://user-images.githubusercontent.com/7074/89212845-7e731900-d5c4-11ea-9529-235f1ca4d925.gif)